### ISSUE TITLE
[Accessibility] Semantic heading tag is missing in issue and PR webview comments

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode-pull-request-github#5524]
Type: <b>Bug</b>

CC @meganrogge 

If screen reader users open any GitHub issue or PR containing multiple comments in VSCode, SR users expect to jump between comments by using semantic heading. On gitHub webpage, each commentor name is denoted with h3 tag. However, currently, VSCode GH extension does not have the h3 tag for commentors name so it's hard to navigate between comments.

Extension version: 0.77.2023112911
VS Code version: Code - Insiders 1.85.0-insider (8762127fc97b8aaae5f3a0dd6a6d853d9a1b7574, 2023-11-29T17:38:23.440Z)
OS version: Windows_NT x64 10.0.22631
Modes:

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|11th Gen Intel(R) Core(TM) i5-1145G7 @ 2.60GHz (8 x 2611)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: enabled_on<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: enabled|
|Load (avg)|undefined|
|Memory (System)|15.71GB (5.82GB free)|
|Process Argv|--crash-reporter-id b05b88e5-8894-4031-ae34-fa034ebddea9|
|Screen Reader|yes|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368:30146709
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
vsaa593:30376534
pythonvs932:30404738
py29gd2263:30784851
vsclangdf:30492506
c4g48928:30535728
dsvsc012:30540252
pynewext54:30618038
a9j8j154:30646983
showlangstatbar:30737417
pythonfmttext:30716741
fixshowwlkth:30771523
showindicator:30805243
pythongtdpath:30726887
i26e3531:30792625
welcomedialog:30812478
pythonnosmt12:30779711
pythonidxpt:30768918
pythonnoceb:30776497
asynctok:30898717
dsvsc013:30777762
dsvsc014:30777825
pythonmpswarning:30901777
dsvsc015:30821418
pythontestfixt:30866404
pythonregdiag2:30902439
pyreplss1:30879911
pythonmypyd1:30859725
pythoncet0:30859736
pythontbext0:30879054
accentitlementst:30870582
dsvsc016:30879898
dsvsc017:30880771
dsvsc018:30880772
aa_t_chat:30882232

```

</details>

**Comment by alexr00** [comment]
This is cool. I didn't know screen readers did this.

**Comment by rzhao271** [comment]
In the comment-header-info component, it looks like the author name is denoted with a strong tag instead of an h3 tag. Should that be changed as well?

![Devtools showing that in-file PR review comments have the author bolded with a strong tag inside of a div rather than the title section being inside an h3](https://github.com/microsoft/vscode-pull-request-github/assets/7199958/04f70377-dec0-4fd8-af61-7ff2a3222d49)

**Comment by jooyoungseo** [comment]
Strong tag is not a semantic tag. It’s rather a formatting tag. For screen reader users, please replace it with h3 semantic tag.

**Comment by alexr00** [comment]
For comment threads in an the editor we have commands with keybindings: `editor.action.nextCommentThreadAction` and `editor.action.previousCommentThreadAction` (`alt+f9` and `shift+alt+f9` respectively). Are semantic tags also supposed to work in the editor?
